compiler/rustc_expand/src/expand.rs

```diff
@@ -1299,36 +1307,37 @@ impl<'a, 'b> MutVisitor for InvocationCollector<'a, 'b> {
 
                         *mod_kind = ModKind::Loaded(krate.items, Inline::No, inner_span);
                         item.attrs = krate.attrs;
-                        // File can have inline attributes, e.g., `#![cfg(...)]` & co. => Reconfigure.
-                        item = match self.configure(item) {
-                            Some(node) => node,
-                            None => {
-                                if *pushed {
-                                    sess.included_mod_stack.borrow_mut().pop();
-                                }
-                                return Default::default();
-                            }
-                        };
                         dir
                     }
                 };
 
-                // Set the module info before we flat map.
-                self.cx.current_expansion.directory_ownership = ownership;
-                module.directory = path;
-                module.mod_path.push(ident);
-                let orig_module =
-                    mem::replace(&mut self.cx.current_expansion.module, Rc::new(module));
-
-                let result = noop_flat_map_item(item, self);
+                if item.attrs.len() > old_attrs_len {
```

Comment:
This can only happen in the `ModKind::Unloaded` case, right? Could this be moved into that match arm?